_posts/2017-07-25-Creating-reactive-calendar-in-angular4.md

```diff
@@ -100,33 +100,33 @@ const firebaseConfig = {
 };
 ```
 
-Let's continue, start the project by running the following command and open your browser on [http://localhost:4200](http://localhost:4200). 
+Let's continue. Start the project by running the following command and open your browser on [http://localhost:4200](http://localhost:4200). 
 
 
 ```
 npm start
 ```
 
-As you can see this just handles static data, the buttons/inputs won't work and the appointments are not loaded yet.
+As you can see, this just handles static data, the buttons/inputs won't work, and the appointments are not loaded yet.
 This is where we start from.
 
-## Thinking reactive
+## Thinking Reactively
 
-Here comes the tricky part, we are trying to forget imperative programming for now, and we are trying to evolve into a reactive mind-set.
+Now comes the tricky part. We are trying to forget imperative programming for now, and we are trying to evolve into a reactive mindset.
 
-### Marble diagrams
+### Marble Diagrams
 
-To be able to think reactive we need some kind of graphical model so we can picture streams in our head. Marble diagrams are a great way to do this.
-As we can see in the image below, a marble represents a value over time.
+To be able to think reactively, we need some kind of graphic model so we can picture streams in our head. Marble diagrams are a great way to do that.
+As you can see in the image below, a marble represents a value over time.
```

Comment:
I try to keep a we approach, since it's more user friendly. like "We are doing this tutorial together" :)